### DISCUSSION TITLE
What causes a blank SYS: window?

### DISCUSSION THREAD

**Discussion by hayai-M5** [thehighnibble/thehighnibble.github.io#151] [IMSAI8080esp replica]
I've been having this problem intermittently, but now it seems permanent. I successfully upgraded to the v1.12.0 release (from v1.10.0), but this didn't fix the problem. The rest of the window functionality is there - I can reboot, edit config files, resize the window (nice v1.12.0 addition!).  Am I mistaken in the belief this window should work with all ROMs? (Currently running Z80 & cp/m 2.2. Also see the problem with the default configuration.)

**Comment by hayai-M5** [comment]
I'll add that this is using both chrome and firefox on macOS. And only one window open to the imsai site. And with the imasi connected to my wifi network (i.e., not in AP mode.)

**Comment by daniel-mcclain** [comment]
i have the same issue

**Comment by dmcnaugh** [comment]
The problem is likely caused by a value in your `boot.conf` that breaks the syntax of the JSON that is transmitted to populate the `SYS:` window.

The most likely candidate is your WiFi password contains punctuation that upsets JSON - likely a `"` or a `'`

You can check by pointing your browser at `http://imsai8080/system` and you should see the JSON in question and may be able to see where the syntax rules are broken.

The fault is mine because I do not check for this and encode the values to protect the JSON syntax. I will try to fix this in the next release.

Meanwhile the only solution would be to change the value in your `boot.conf` that is upsetting it, but you may not want to do this because it might force you to change your WiFi password.

**Comment by dmcnaugh** [comment]
This has been fixed in the latest firmware release see #169
